csrankings-l.csv

```diff
@@ -827,6 +827,7 @@ Luís F. M. de Moraes,UFRJ,http://www.cos.ufrj.br/~moraes,NOSCHOLARPAGE
 Luis Felipe Gonzalez,Queensland University of Technology,https://research.qut.edu.au/qcr/people/felipe-gonzalez,c_5ewxYAAAAJ
 Luís Felipe M. de Moraes,UFRJ,http://www.cos.ufrj.br/~moraes,NOSCHOLARPAGE
 Luís G. Rueda,University of Windsor,http://www.uwindsor.ca/science/computerscience/1033/dr-luis-rueda,2OPjLFkAAAAJ
+Luis Garcia,University of Utah,https://lagarcia.us/,F6Gzg9gAAAAJ

```

Comment:
Disambiguation entry (see https://dblp.org/pid/67/136). Please remove and follow instructions in the contribution guidelines.

Reply:
As requested, removed Luis because of the DBLP entry issue.